WooCommerce/WooCommerceScreenshots/WooCommerceScreenshots.swift

```diff
@@ -135,22 +147,29 @@ extension BaseScreen {
         let mode = XCUIDevice.inDarkMode ? "dark" : "light"
         let filename = "\(screenshotCount)-\(mode)-\(title)"
 
-        snapshot(filename)
+        snapshot(filename, timeWaitingForIdle: 0)
 
         return self
     }
 }
 
 extension ScreenObject {
 
+    @discardableResult
+    func lockScreen() -> Self {
+        XCUIDevice.shared.perform(NSSelectorFromString("pressLockButton"))
```

Comment:
does this usage have an official doc? I hope this doesn't break in future iOS versions 😅